apps/framework-cli/src/cli/commands.rs

```diff
@@ -85,6 +87,8 @@ pub enum Commands {
         #[arg(short, long, default_value = "false")]
         streaming: bool,
     },
+
+    Metrics {},
```

Comment:
We can add a quick description of this command using `/// <text>`. It will show up if they do `moose-cli help` or `moose-cli metrics help`